frontend/server/src/main/java/org/pytorch/serve/metrics/configuration/YAMLMetricConfigurationHandler.java

```diff
@@ -0,0 +1,25 @@
+package org.pytorch.serve.metrics.configuration;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.composer.ComposerException;
+import org.yaml.snakeyaml.constructor.Constructor;
+
+public final class YAMLMetricConfigurationHandler {
+    private YAMLMetricConfigurationHandler() {
+        throw new UnsupportedOperationException();
+    }
+
+    static MetricConfiguration loadConfiguration(String configFilePath)
+            throws FileNotFoundException, ComposerException, RuntimeException {
+        Constructor constructor = new Constructor(MetricConfiguration.class);
+        Yaml yaml = new Yaml(constructor);
+        FileInputStream inputStream = new FileInputStream(new File(configFilePath));
+        MetricConfiguration config = yaml.load(inputStream);
+        config.validate();
+
+        return config;
+    }
+}
```

Comment:
Only a simple function as following is needed.

public static <T> T readYamlFile(File file, Class<T> type)
            throws InvalidModelException, IOException {
        Yaml yaml = new Yaml(new Constructor(type));
        try (Reader r =
                new InputStreamReader(
                        Files.newInputStream(file.toPath()), StandardCharsets.UTF_8)) {

            return yaml.load(r);
        } catch (YAMLException e) {
            throw new InvalidModelException("Failed to parse model config yaml file.", e);
        }
    }